packages/flight-icons/catalog.json

```diff
@@ -731,6 +1163,42 @@
       "width": 16,
       "height": 16
     },
+    {
+      "id": "2372:404",
+      "fileName": "lightlytics-color-24",
+      "iconName": "lightlytics-color",
+      "description": "lightlytics",
+      "size": "24",
+      "width": 24,
+      "height": 24
+    },
+    {
+      "id": "2372:406",
+      "fileName": "lightlytics-color-16",
+      "iconName": "lightlytics-color",
+      "description": "lightlytics",
+      "size": "16",
+      "width": 16,
+      "height": 16
+    },
+    {
+      "id": "2372:411",
+      "fileName": "lightlytics-color-24",
+      "iconName": "lightlytics-color",
+      "description": "lightlytics",
+      "size": "24",
+      "width": 24,
+      "height": 24
+    },
+    {
+      "id": "2372:416",
+      "fileName": "lightlytics-color-16",
+      "iconName": "lightlytics-color",
+      "description": "lightlytics",
+      "size": "16",
+      "width": 16,
```

Comment:
These are double, it means there are two icons with the same name in the Figma file.

Reply:
I fixed this. They are different, it was the name that was duplicated.

Reply:
I think there is still something not working correctly. I see that only the color version assets are generated, not the mono ones:
<img width="297" alt="screenshot_1706" src="https://user-images.githubusercontent.com/686239/181094589-cd49481a-5fdb-444e-8e1c-6abc251a4b75.png">

Reply:
The library was probably not published yet. It is now.